### PR TITLE
Test: Add test base for TPT in InheritanceQueryTestBase

### PR DESCRIPTION
**Pull request by smitpatel**
Part of #2266

### CODE REVIEW

File: test/EFCore.Relational.Specification.Tests/Query/TPTQueryRelationalFixture.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.EntityFrameworkCore.TestModels.Inheritance;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+
+namespace Microsoft.EntityFrameworkCore.Query
+{
+    public abstract class TPTQueryRelationalFixture : InheritanceQueryFixtureBase
```
Comment (AndriySvyryd) [inline test/EFCore.Relational.Specification.Tests/Query/TPTQueryRelationalFixture.cs:9]:
```suggestion
    public abstract class TPTRelationalInheritanceQueryFixture : InheritanceQueryFixtureBase
```

Comment (smitpatel) [reply test/EFCore.Relational.Specification.Tests/Query/TPTQueryRelationalFixture.cs:9]: But we also have https://github.com/dotnet/efcore/blob/master/test/EFCore.Relational.Specification.Tests/Query/InheritanceRelationalFixture.cs right now (which is getting renamed in this PR).
So do Inheritance comes first or Relational?

Comment (smitpatel) [reply test/EFCore.Relational.Specification.Tests/Query/TPTQueryRelationalFixture.cs:9]: Also https://github.com/dotnet/efcore/tree/master/test/EFCore.Relational.Specification.Tests/Query

Comment (AndriySvyryd) [reply test/EFCore.Relational.Specification.Tests/Query/TPTQueryRelationalFixture.cs:9]: TPTInheritanceQueryFixture.
Relational is implied

**Pull request merged**